contrib/destination-starrocks/src/main/java/io/airbyte/integrations/destination/starrocks/DefaultStreamLoader.java

```diff
@@ -153,27 +98,27 @@ public StreamLoadResponse send(List<AirbyteRecordMessage> records) throws Except
 
         int port = properties.getHttpPort();
         String scheme = "http";
-
-        // check if port for approapriate url scheme
-        if (port == 443) {
+        // check if port for appropriate url scheme
+        if (properties.getSSL()) {
             scheme = "https";
         }
 
         String sendUrl = String.format("%s://%s:%d%s", scheme, host, port, loadUrlPath);
-        String label = StreamLoadUtils.label(loadTable);
+        LOG.info("Stream Load URL : {}", sendUrl);
```

Comment:
You can remove the log in the formal code, just for debugging.

Reply:
Alright, it is gone now